crates/extensions/aws_mapper_ext/src/converter.rs

```diff
@@ -49,29 +41,30 @@ impl AwsConverter {
     }
 }
 
-#[async_trait]
+pub fn make_valid_topic_or_panic(topic_name: &str) -> Topic {
```

Comment:
You could drop this function completely as `Topic::new_unchecked()` already exists in `mqtt_channel`.